db/python/layers/seqr.py

```diff
@@ -392,71 +411,75 @@ async def update_es_index(
         fn_path = os.path.join(SEQR_MAP_LOCATION, filename)
         # pylint: disable=no-member
 
+        with AnyPath(fn_path).open('w+') as f:  # type: ignore
+            f.write('\n'.join(rows_to_write))
```

Comment:
My gut is we should do this _per_ index, or not once for all? I'm less clear on how strict seqr wants this file, but would've thought seqr will try to link all people listed against the index. Or at the very least if this isn't true, can you add a comment RE this.